README.md

```diff
@@ -31,6 +31,38 @@ initialization
   GetParamManager.RegisterParam('MyUserSpecificParam', 'DefaultValue', ssUser, psRemote);
 ```
 
+### Scopes
+
+The param scope when register will dictate how the param values are separated between users or companies.
+
+**ssGlobal**
+
+Every user in every company can read and write it's value. In the database the param will be stored like this:
```

Comment:
What do you think of:

"This param value will be shared between any user in any company."

I think the "...can read and write.." sounds like some sort of access rights and may confuse some people.